TestControllers/AzureController.psm1

```diff
@@ -87,15 +87,25 @@ Class AzureController : TestController
 			$secrets = $this.XMLSecrets.secrets
 			$azureConfig.Subscription.SubscriptionID = $secrets.SubscriptionID
 			$azureConfig.TestCredentials.LinuxUsername = $secrets.linuxTestUsername
-			$azureConfig.TestCredentials.LinuxPassword = $secrets.linuxTestPassword
-			$azureConfig.ResultsDatabase.server = $secrets.DatabaseServer
-			$azureConfig.ResultsDatabase.user = $secrets.DatabaseUser
-			$azureConfig.ResultsDatabase.password = $secrets.DatabasePassword
-			$azureConfig.ResultsDatabase.dbname = $secrets.DatabaseName
+			$azureConfig.TestCredentials.LinuxPassword = if ($secrets.linuxTestPassword) { $secrets.linuxTestPassword } else { "" }
+			$azureConfig.ResultsDatabase.server = if ($secrets.DatabaseServer) { $secrets.DatabaseServer } else { "" }
```

Comment:
Those 4 line changes will need to follow up the rest of the code change. If I understand correct, this should allow the secret xml file contains none of DB information. So we should add 'filtering' lines in each SQL statement in the LISAv2. It's not about this PR, but recommend to add a new issue for future followup.